moveit_planners/pilz_industrial_motion_planner/include/pilz_industrial_motion_planner/joint_limits_interface_extension.h

```diff
@@ -36,63 +36,306 @@
 #define JOINT_LIMITS_INTERFACE_EXTENSION_H
 
 #include "pilz_industrial_motion_planner/joint_limits_extension.h"
-#include <joint_limits_interface/joint_limits_rosparam.h>
+#include <limits>
 
-namespace pilz_industrial_motion_planner
-{
+// TODO(henning): Re-include when this is available, until then the headers content is copied below
+// TODO(sjahr): Current implementation does not offer the desired API (08/2021). Use draft from dstogl instead
+// (https://github.com/ros-controls/ros2_control/pull/462/files) #include <joint_limits_interface/joint_limits_rosparam.hpp>
+//////////////////////////////////////////////////////////////
+// start of <joint_limits_interface/joint_limits_rosparam.hpp>
 namespace joint_limits_interface
 {
+inline bool declare_parameters(const std::string& joint_name, const rclcpp::Node::SharedPtr& node)
+{
+  const std::string param_base_name = "joint_limits." + joint_name;
```

Comment:
@sjahr I had to add the namespace (in this case "robot_description_planning") to the param_base_name to get it running. Same for three other occurences in this file.